src/components/LayoutComponents/HeaderAndFooterLayoutComponent/HeaderAndFooterLayoutComponent.js

```diff
@@ -0,0 +1,17 @@
+import classes from "./HeaderAndFooterLayoutComponent.module.scss";
```

Comment:
HeaderAndFooterLayoutComponent - putting this component inside a components folder, we know that this is a component so put the naming back to HeaderAndFooterLayout